codalab/lib/bundle_cli.py

```diff
@@ -1069,9 +1069,10 @@ def do_ls_command(self, args):
         return self.create_structured_info_map([('refs', reference_map)])
 
     def _worksheet_description(self, worksheet_info):
-        return '### Worksheet: %s\n### Title: %s\n### Owner: %s(%s)\n### Permissions: %s%s' % \
+        return '### Worksheet: %s\n### Title: %s\n### Tags: %s\n### Owner: %s(%s)\n### Permissions: %s%s' % \
```

Comment:
perhaps for a future diff: this substitution starting to get unwieldy, it might be preferable to use `str.format`.